src/main/java/com/solvd/Main.java

```diff
@@ -185,7 +189,14 @@ public static void userActions(Scanner scanner, Client client) throws EntityNotF
                     deleteAccount(scanner, client);
                     break;
                 case "7":
-                    System.out.println(AGREEMENT_SERVICE.getByClientId(client.getId()));
+                    Optional<Agreement> agreement = AGREEMENT_SERVICE.getByClientId(client.getId());
+                    if (agreement.isPresent()) {
+                        System.out.println(Agreement.getTableHeader());
+                        System.out.println(agreement);
```

Comment:
System.out.println(Optional<Agreement>) instead of System.out.println(Agreement)